client/src/modules/chapters/pages/chapterPage.tsx

```diff
@@ -27,11 +27,32 @@ export const ChapterPage: NextPage = () => {
 
   return (
     <VStack>
-      <Heading>{data.chapter.name}</Heading>
-      <Text>{data.chapter.description}</Text>
-
-      <Heading size="md">Events:</Heading>
-      <VStack w={['60%', '90%', '60%']} maxW="800px">
+      <Stack w={['90%', '90%', '60%']} maxW="600px" spacing={6} mt={10} mb={5}>
+        <Image
+          boxSize="100%"
+          maxH="300px"
+          src={data.chapter.image}
+          alt="The freeCodeCamp logo"
```

Comment:
Same as https://github.com/freeCodeCamp/chapter/pull/652/ we should add the alt text with the image url

Reply:
It seems like on #652 we are setting alt to an empty string. Other than the alt, it is LGMT.

Reply:
```suggestion
          alt=""
```